maxim/models/dataset.py

```
@@ -85,5 +85,5 @@ def to_json(self):
 
 DataValue = list[T]
 
-ManualData = Dict[str, Union[str, List[str], None]]
-Data = Union[str, List[ManualData], ManualData, Callable[[int], Optional[ManualData]]]
+LocalData = Dict[str, Union[str, List[str], None]]
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)